incompatibility/incomp-scanning_test.go

```diff
@@ -87,6 +90,86 @@ func TestIncompatibilityExistence(t *testing.T) {
 	}
 }
 
+// Test verifying detailing proces of an incompatibility report
+func TestDetailing(t *testing.T) {
+	var detailingTest = []struct {
+		baseReport *IncompatibilityReport
+	}{
+		{createReport(t, "../examples/petstore/petstore.yaml")},
```

Comment:
Using a helper method from another test files is a bit confusing.   Can we use the public: GnosticIncompatibiltyScanning to generate the report to compare against?

Reply:
tests now using GnosticIncompatibiltyScanning but with a helper function that handles error and gets report from env object